scripts/legacy/tar/linux/opensearch-tar-install.sh

```diff
@@ -40,33 +40,25 @@ if ! grep -q '## OpenSearch Performance Analyzer' $OPENSEARCH_HOME/config/jvm.op
 fi
 echo "done plugins"
 
-##Check KNN lib existence in OpenSearch TAR distribution
-echo "Checking kNN library"
-FILE=`ls $KNN_LIB_DIR/lib*.so`
-if test -f "$FILE"; then
-    echo "FILE EXISTS $FILE"
-else
-    echo "TEST FAILED OR FILE NOT EXIST $FILE"
-fi
-
 ##Set KNN Dylib Path for macOS and *nix systems
 if echo "$OSTYPE" | grep -qi "darwin"; then
     if echo "$JAVA_LIBRARY_PATH" | grep -q "$KNN_LIB_DIR"; then
-        echo "KNN lib path has been set"
+        echo "k-NN libraries found in JAVA_LIBRARY_PATH"
```

Comment:
Could you update the messages to include the paths that were used after the variable name?  Would that help with debugging?

Reply:
In my opinion, I think if it is found, just saying so is okay.

If it is not found, echo'ing the updated path could be useful. Currently, we do echo the path. But, do you mean update it to something like this? 
```
        echo "k-NN libraries not found in JAVA_LIBRARY_PATH. Updating path to: $JAVA_LIBRARY_PATH."
```